network/src/noise/handshake.rs

```diff
@@ -693,8 +693,8 @@ mod test {
         client.network_context = NetworkContext::mock_with_peer_id(PeerId::random());
         let (client_res, server_res) = perform_handshake(&client, &server, server_public_key);
 
-        trace!("client_res: {}", client_res.unwrap_err());
-        trace!("server_res: {}", server_res.unwrap_err());
+        client_res.unwrap_err();
+        server_res.unwrap_err();
```

Comment:
This test doesn't check the error?

Also, let's stick to docs, or code fixes at once

Reply:
`unwrap_err` will panic if it's not an `Err` : )